Update reference-collation-types.md

TsunehisaYamasaki opened the pull request:
This limitation is confirmed by these ICM and work item. ICM: https://portal.microsofticm.com/imp/v5/incidents/details/545458083/impact Work Item: https://msdata.visualstudio.com/A365/_workitems/edit/3553326

TsunehisaYamasaki committed:
Update reference-collation-types.md

This limitation is confirmed by these ICM and work item.
ICM: https://portal.microsofticm.com/imp/v5/incidents/details/545458083/impact
Work Item: https://msdata.visualstudio.com/A365/_workitems/edit/3553326

WilliamDAssafMSFT commented on articles/synapse-analytics/sql/reference-collation-types.md:
```diff
@@ -28,7 +28,7 @@ You can specify the default serverless SQL pool database collation at creation t
 
 > [!NOTE]
-> Collation cannot be changed on an existing dedicated SQL pool database. If you need to have a different collation at the dedicated SQL pool level, create a new dedicated SQL pool with the required collation.
+> Collation cannot be changed on an existing dedicated SQL pool database. If you need to have a different collation at the dedicated SQL pool level, create a new dedicated SQL pool with the required collation. We can change Collation from Azure Portal. but, we can't change Collation from Synapse Studio. it's limitation for now.
```
```suggestion
> Collation cannot be changed on an existing dedicated SQL pool database. If you need to have a different collation at the dedicated SQL pool level, create a new dedicated SQL pool with the required collation. To specify a non-default collation when creating a dedicated SQL pool, use the Azure portal.
```

TsunehisaYamasaki committed:
Update articles/synapse-analytics/sql/reference-collation-types.md

Co-authored-by: William Assaf MSFT <74387232+WilliamDAssafMSFT@users.noreply.github.com>

TsunehisaYamasaki committed:
Update reference-collation-types.md

We confirmed T-SQL and PowerShell could create a pool with a non-default collation.

TsunehisaYamasaki committed:
Update reference-collation-types.md
